cypress/integration/examples/connectors.spec.js

```diff
@@ -0,0 +1,97 @@
+/// <reference types="cypress" />
+
+context('Connectors', () => {
+  beforeEach(() => {
```

Comment:
The test suite is not related to the task at hand. You should create a new file and write tests for the Book Store app following the given task description.